### PR TITLE
ODC-5013 - Epic Automation

### PR DESCRIPTION
**Pull request by gajanan-more**
**Fix:** https://issues.redhat.com/browse/ODC-5864

**Problem:** Automation to verify new changes made in the Add page

**Solution:** This script will verify all the new changes made in the Add page because of epic [ODC-5013](https://issues.redhat.com/browse/ODC-5013)

**Test Setup:**
1. Replace `"TAGS": "@smoke and not @manual and not @to-do",` with `"(@smoke or @regression) and not (@manual or @to-do)",` in devconsole/cypress.json under env
2. Run `yarn run test-cypress-devconsole` in `frontend`
3. Execute feature file: `frontend/packages/dev-console/integration-tests/features/addFlow/add-page.feature`

Test Execution Screenshot:
Screenshot from 2021-06-07 15-09-50

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/addFlow/add-page.feature [hunk 1]
```diff
@@ -5,89 +5,82 @@ Feature: Add page on Developer Console
...
+        @smoke
+        Scenario: Hide Getting Started Resources from View: A-11-TC06
+            Given getting started resources is not hidden from view
+             When user selects Hide from view option from kebab menu
+             Then user will not see Getting started resources card
+
+
+        @smoke
+        Scenario: Show Getting Started Resources: A-11-TC07
+             When user clicks on Show getting started resources link
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/addFlow/add-page.feature:59]: Given: user has hidden Getting Started Resources from View

Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/addFlow/add-page.feature:59]: Updated

**Comment by gajanan-more** [comment]
/remove-component/pipelines

**Comment by gajanan-more** [comment]
Getting started resources card won't be visible on the Console UI if someone closed the link, we can get it back by clearing local storage on the localhost but it's not possible on Console UI. Raised one bug for that: https://issues.redhat.com/browse/ODC-5942

**Comment by makambalaji** [comment]
Verified on Remote cluster, except the issue mentioned by gaja. Remaining everything looks good
/lgtm
/approve

**Pull request merged**